lib/xandra/cluster/control_connection.ex

```diff
@@ -246,7 +246,7 @@ defmodule Xandra.Cluster.ControlConnection do
 
       data = update_in(data.lbp, &LoadBalancingPolicy.update_host(&1, host, :added))
       send(data.cluster, {:host_added, new_host})
-      put_in(data.peers[{new_host.address, new_host.port}], %{status: :up, host: host})
+      put_in(data.peers[{new_host.address, new_host.port}], %{status: :up, host: host, last_seen_at:  System.system_time(:millisecond)})
```

Comment:
I think the new `last_seen_at` is helpful, but can we call it something like `last_discovered_at`? This is effectively what's happening here: the node was *discovered* in the `peers` table. I think the term "seen" is too generic and might be confusing. Wdyt?

Reply:
sure, I've got no problems with whatever the name is.